bascule/basculehttp/constructor.go

```diff
@@ -69,7 +70,8 @@ func (c *constructor) decorate(next http.Handler) http.Handler {
 				Token:         token,
 			},
 		)
-		logger.Log(level.Key(), level.DebugValue(), "msg", "authentication added to context", "request", request,
+		logger.Log(level.Key(), level.DebugValue(), "msg", "authentication added to context",
+			"request headers", request.Header, "request URL", request.URL,
```

Comment:
Leave out anything related to the request.  Let your caller set that up in the `getLogger` function.